public/API/API_GetClaims.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Validation\Rule;
+use LegacyApp\Community\Enums\ClaimFilters;
+
+/*
+ *  API_GetClaims - returns information about 1000 max set claims.
```

Comment:
Which 1000? I assume the latest 1000?

Reply:
Addressed in https://github.com/RetroAchievements/RAWeb/pull/1435/commits/b3e44d98130302069814d21d7e7cdc790c808ab0